.kres.yaml

```diff
@@ -96,6 +100,34 @@ spec:
         @$(foreach target,$(NONFREE_TARGETS),echo $(REGISTRY)/$(USERNAME)/$(target):$(shell $(ARTIFACTS)/bldr eval --target $(target) --build-arg TAG=$(TAG) '{{.VERSION}}' 2>/dev/null) >> _out/extensions-metadata;)
 ---
 kind: custom.Step
+name: extensions-info
+spec:
+  makefile:
+    enabled: true
+    phony: true
+    depends:
+      - $(ARTIFACTS)/bldr
+    script:
+      - |
+        @find ./ -name "manifest.yaml" -print0 | env LC_ALL=en_US sort -z | xargs -r0 -I{} sh -c 'echo "---\\n$$(cat {})"' > extensions.yaml
+---
+kind: custom.Step
+name: check-dirty
```

Comment:
I feel like `check-dirty` should be implemented as a base step in our Kres flow, as it would work for almost any project, if we hook it up after `generate` (or could be even the last step in the `generate` sequence).

Reply:
well, we can't really add it to the `generate` step, as it's either go or js generate steps.
While this particular project is neither of those.
So we need to add it to some common part for all, but after we detect the project type.

Reply:
yep, makes sense